README.md

```diff
@@ -1,2 +1,56 @@
 # publisher-frontend
 Publisher frontend to manage course creation and marketing content curation.
+
+# Important Note
+
+This repository is in early Alpha stages of development and will be for the foreseeable future. Until then it is 
+not recommended for use in production.
+
+# Development Environment
+
+## Getting Started
+
+Run the following commands to get started with the Publisher Development environment.
+
+1. Install Docker locally. 
+    * Docker is used to generate a local development environment that contains all the required tools and libraries to 
+    develop against. 
+    
+    * The most recent stable version of Docker should work.
+    
+2. Run the following docker-compose commands to get started.
+
+    ```
+    docker-compose build
+    docker-compose up 
```

Comment:
I might replace this with "make build up"

Reply:
I am really torn about the Makefile approach.

On one side it seems unnecessary for the docker commands.  Especially the simple commands like docker-compose. 

On the other side it is nice for the more complex commands.

Reply:
Updated to use make